homeassistant/components/vicare/strings.json

```
@@ -9,13 +9,23 @@
           "password": "[%key:common::config_flow::data::password%]",
           "client_id": "Client ID",
           "heating_type": "Heating type"
+        },
+        "data_description": {
+          "username": "The email address to login to your ViCare account.",
+          "password": "The password to login to your ViCare account.",
+          "client_id": "The ID of the API client created in the Viessmann developer portal.",
+          "heating_type": "Use to overrule the device auto detection."
```

```suggestion
          "heating_type": "Allows to overrule the device auto detection."
```
otherwise sounds strange imo